internal/constellation/applyinit.go

```diff
@@ -99,6 +101,29 @@ func (a *Applier) Init(
 	return doer.resp, nil
 }
 
+// RewrittenKubeconfigBytes returns the kubeconfig bytes with the cluster server address rewritten to the given endpoint.
+// This is usually used to rewrite the cluster endpoint to a public endpoint in case a private LB is used.
+func (a *Applier) RewrittenKubeconfigBytes(kubeconfigBytes []byte, clusterEndpoint string) ([]byte, error) {
```

Comment:
I also thought about this, but I thought it might be semantically unclear if we did not return the actual init response as part of the method. If we only return the stuff we actually need, we can do this though